interview-questions.md

```diff
@@ -4,39 +4,39 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Why would you use the method super()?
 
-  Your answer:
+  Your answer:you would use it to call the base classe's constructor
 
   Researched answer:
 
 
 
 2. What is a virtual DOM?
 
-  Your answer:
+  Your answer:its the representation of the UI
```

Comment:
Totally, but I'd want to hear a bit more about this. Don't be afraid to expand upon your answer!